src/LazyRendering/Http/StreamedController.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace App\LazyRendering\Http;
+
+use App\Blog\Entity\Post;
+use App\Blog\Post\PostRepository;
+use App\Blog\Widget\PostCard;
+use App\LazyRendering\View\MainLayout;
+use Cycle\ORM\ORMInterface;
+use Psr\Http\Message\ResponseInterface;
+use RuntimeException;
+use Yiisoft\Html\Html;
+
+class StreamedController extends BaseController
+{
+    public const ROUTE_NAME = 'lazy-render';
+
+    protected $pageLayout = MainLayout::class;
+
+    public function pageIndex(): iterable
+    {
+        foreach (get_class_methods($this) as $method) {
```

Comment:
That's way too magical to understand it. How about calling all these methods explicitly for demo purpose instead?